services/apps/script_executor_worker/src/activities/common.ts

```diff
@@ -14,9 +15,8 @@ import { svc } from '../main'
 export async function mergeMembers(
   primaryMemberId: string,
   secondaryMemberId: string,
-  tenantId: string,
 ): Promise<void> {
-  const url = `${process.env['CROWD_API_SERVICE_URL']}/tenant/${tenantId}/member/${primaryMemberId}/merge`
+  const url = `${process.env['CROWD_API_SERVICE_URL']}/tenant/${DEFAULT_TENANT_ID}/member/${primaryMemberId}/merge`
```

Comment:
We no longer have tenantId in this url, we can remove that part and call it like `${process.env['CROWD_API_SERVICE_URL']}/member/${primaryMemberId}/merge`